setup.cfg

```diff
@@ -19,38 +19,10 @@ classifiers =
     Topic :: System :: Filesystems
```

Comment:
I believe this packaging information is for setuptools, so should be migrated to poetry's pyproject.toml